### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by kevinsuedmersen** [keras-team/keras#16668]
Feature request to add a boolean parameter in `tf.keras.models.model_clone` (https://github.com/keras-team/keras/blob/07e13740fd181fc3ddec7d9a594d8a08666645f6/keras/models/cloning.py#L380-L449) that will allow us to use the same weights of the to-be-cloned model in the new, cloned model instance.

**Comment by mattdangerw** [comment]
There is already a pretty simple way to share weights after cloning.

`cloned_model.set_weights(original_model.get_weights())`

Would that work for your use case?

**Comment by kevinsuedmersen** [comment]
Yes, you are right and we ended up doing what you suggested. I guess I was a little bit confused when I noticed that the trained weights were not copied/cloned. But since this is actually mentioned in the documentation, I think we can close this feature request.
